README.md

```diff
@@ -1,9 +1,20 @@
-## MeshWalk.js
+# MeshWalk.js
 
 MeshWalk.js is a JS library which helps your TPS game development with three.js.
 
-examples
+## Version Compatability
+
+THREE.js [r88 development](https://github.com/mrdoob/three.js/blob/2b1a620308ce4cd3388a840cc291773a74cfa52f/build/three.min.js)
+
+BLUEBIRD.js [3.5.0](https://cdn.jsdelivr.net/bluebird/latest/bluebird.min.js)
+
+`THREE.r74+ EventDispatcher.prototype.apply() changed to: EventDispatcher.apply()`
+
+`THREE.r84+ EventDispatcher.apply has been removed. Inherit or Object.assign the prototype to mix-in instead.`
 
+`THREE.r88 dev Compatible! :)`
+
```

Comment:
I don't care older version compatibility for three.js. Could we remove this section?

Reply:
Sure yeah, I was just writing that in case anyone has something in between 73-88 but those are obsolete with someone working on the latest version of threes and should update.